solana/README.md

```diff
@@ -103,5 +106,42 @@ In order to run them, the Solana programs must be built and their IDL made avail
 To ensure the SDK has the generated IDL, run the tests with the make command:
 
 ```sh
-make anchor-test
-```
\ No newline at end of file
+make test
+```
+
+### Troubleshooting
+
+<details>
+<summary><code>make: *** No rule to make target `test'.  Stop.</code></summary>
+
+- Ensure `Makefile` has target `test`
```

Comment:
not sure if this should be called out here, the makefile does have a test target. How would someone hit this issue?

Reply:
I think I hit the issue because the README was not up-to-date with the changes that were being made. Currently, this issue would not be hit.